src/objprint/frame_analyzer.py

```diff
@@ -102,3 +102,27 @@ def _splitlines_no_ff(self, source: str) -> List[str]:  # pragma: no cover
         if next_line:
             lines.append(next_line)
         return lines
+
+    def return_object(self, frame: Optional[FrameType]) -> bool:
+        if frame is None:
+            return True
+        try:
+            current_frame: Optional[FrameType] = frame
```

Comment:
I don't think this hint is needed as `frame` is already hinted?

Reply:
I think this type hint would be necessary, as without it the below lint error would occur:
```
src/objprint/frame_analyzer.py:115: error: Incompatible types in assignment (expression has type "Optional[FrameType]", variable has type "FrameType")
Found 1 error in 1 file (checked 5 source files)
```

Reply:
Ah, because you ruled out the `None` case for `frame` above, okay.